my_list.rb

```diff
@@ -0,0 +1,17 @@
+class MyList
+  include MyEnumerable
```

Comment:
- [ ] "MyEnumerable" needs to be called as "require_relative './my_enumerable.rb' at the top to be capable to use the methods" 😉